### PR TITLE
Update servers.pvsc and fix to configure remote rendering

### PR DESCRIPTION
**Pull request by estreby**

### CODE REVIEW

File: scripts.d/support/servers.pvsc [hunk 1]
```diff
@@ -12,13 +12,13 @@
           <String default="00:30:00"/>
         </Option>
         <Option label="Number of Nodes:" name="SLURM_NODES" save="true">
-          <Range default="4" max="64" min="1" step="1" type="int"/>
+          <Range default="2" max="2" min="1" step="1" type="int"/>
```
Comment (TurboGit) [inline scripts.d/support/servers.pvsc:15]: The default for our case is 2 max, but is that really the case for all projects?

File: scripts.d/support/install-setup-ng [hunk 2]
```diff
@@ -197,10 +197,26 @@ $SUDO rm -f $prefix/setup
...
 
-# Setup the HPC servers, the template file servers.pvsc
-# will be used by the following script to create or update
-# the file $HOME/.config/ParaView
-./salome_hpc_visu_servers.py
+# Setup the HPC servers, the template file servers.pvsc will be used 
+# by the following script to replace (and copy with date if exist)
+# the file in $HOME/.config/ParaView
+
+local configPath="/home/$USER/.config/ParaView"
```
Comment (TurboGit) [inline scripts.d/support/install-setup-ng:204]: I'm sure you have warnings here as local can only be used inside a function. Here you are in the main part of the script. Can you double check?

File: scripts.d/support/install-setup-ng [hunk 3]
```diff
@@ -197,10 +197,26 @@ $SUDO rm -f $prefix/setup
...
+local userServersPvsc=$configPath/servers.pvsc
+
+if [[ ! -d $configPath ]] ; then
+    mkdir -p $configPath
+fi
+
+diffFile=$(diff $userServersPvsc servers.pvsc)
```
Comment (TurboGit) [inline scripts.d/support/install-setup-ng:211]: easier:

```
diff -q $userServersPvsc servers.pvsc

if [[ $? -ne 0 ]]; then
  ...
```

File: scripts.d/support/install-setup-ng (hunk 2 above)
Comment (TurboGit) [reply scripts.d/support/install-setup-ng:204]: Also please use all lower case for variables (this is the current convention scripts). So:

```
configPath      -> CONFIG_PATH
userServersPvsc -> USER_SERVER_PVSC
```

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some changes, thanks!

File: scripts.d/support/servers.pvsc (hunk 1 above)
Comment (estreby) [reply scripts.d/support/servers.pvsc:15]: I think, It's forbidden to use more than 2 for our group.
I don't know if it's the case in general.

Comment (TurboGit) [reply scripts.d/support/servers.pvsc:15]: So better letting the 64 here. Some other WCKEY group may be allowed to use more nodes. So let's keep 2 as default and 64 as max.

**Comment by estreby** [comment]
Don't already merge this, I have some mistake to compute with master and V9_9_BR on debian9 with cmake taken from native version ... It may not be due to my changes but I prefer to be sure it works before.

File: scripts.d/support/install-setup-ng [hunk 4]
```diff
@@ -197,10 +197,29 @@ $SUDO rm -f $prefix/setup
...
+# the file in $HOME/.config/ParaView
+
+CONFIG_PATH="/home/$USER/.config/ParaView"
+USER_SERVER_PVSC=$CONFIG_PATH/servers.pvsc
+
+if [[ ! -d $CONFIG_PATH ]] ; then
+    mkdir -p $CONFIG_PATH
+fi
+
+diff -q $USER_SERVER_PVSC servers.pvsc
```
Comment (TurboGit) [inline scripts.d/support/install-setup-ng:211]: not needed here, you do that 2 lines below

**Comment by estreby** [comment]
With the 2 last commits

**Comment by TurboGit** [comment]
Manually merged with some squashing of commits to avoid intermediate (and non useful) commits/states.

**Pull request closed without merging**